src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -383,100 +397,281 @@ private SyntaxNode GetNodeContainingTargetNode(SyntaxNode matchingNode)
+                var newTrivia = newArgument.GetLeadingTrivia();
+
+                var oldLeadingWhitespaceTrivia = oldTrivia.TakeWhile(t => t.IsKind(SyntaxKind.WhitespaceTrivia));
+                var newLeadingWhitespaceTrivia = newTrivia.TakeWhile(t => t.IsKind(SyntaxKind.WhitespaceTrivia));
+                var newLeadingNonWhitespaceTrivia = newTrivia.TakeRange(newLeadingWhitespaceTrivia.Count(), newTrivia.Count - 1);
+
+                return newArgument.WithLeadingTrivia(oldLeadingWhitespaceTrivia.Concat(newLeadingNonWhitespaceTrivia));
+            }
 
-            var newTrivia = newArgument.GetLeadingTrivia();
-            var newOnlyHasWhitespaceTrivia = newTrivia.All(t => t.IsKind(SyntaxKind.WhitespaceTrivia));
+            static T ChangeNodeSingleLineCommentsToMultiLine(T param)
+            {
+                var paramTrailingTrivia = param.GetTrailingTrivia();
+
+                var paramSingleLineCommentTrivia = paramTrailingTrivia.Where(trivia => trivia.IsKind(SyntaxKind.SingleLineCommentTrivia)).FirstOrDefault();
+                if (paramSingleLineCommentTrivia != default)
+                {
+                    param = param.ReplaceTrivia(paramSingleLineCommentTrivia, GenerateMultiLineComment(paramSingleLineCommentTrivia));
+                }
+
+                return param;
+            }
 
-            if (oldOnlyHasWhitespaceTrivia && newOnlyHasWhitespaceTrivia)
+            static SyntaxToken ChangeSeparatorSingleLineCommentsToMultiLine(SyntaxToken separator)
             {
-                newArgument = newArgument.WithLeadingTrivia(oldTrivia);
+                var separatorLeadingTrivia = separator.LeadingTrivia;
+                var separatorTrailingTrivia = separator.TrailingTrivia;
+
+                var separatorSingleLineCommentLeadingTrivia = separatorLeadingTrivia.Where(trivia => trivia.IsKind(SyntaxKind.SingleLineCommentTrivia)).FirstOrDefault();
+                if (separatorSingleLineCommentLeadingTrivia != default)
+                {
+                    separator = separator.ReplaceTrivia(separatorSingleLineCommentLeadingTrivia, GenerateMultiLineComment(separatorSingleLineCommentLeadingTrivia));
+                }
+
+                var separatorSingleLineCommentTrailingTrivia = separatorTrailingTrivia.Where(trivia => trivia.IsKind(SyntaxKind.SingleLineCommentTrivia)).FirstOrDefault();
+                if (separatorSingleLineCommentTrailingTrivia != default)
+                {
+                    separator = separator.ReplaceTrivia(separatorSingleLineCommentTrailingTrivia, GenerateMultiLineComment(separatorSingleLineCommentTrailingTrivia));
+                }
+
+                return separator;
             }
 
-            return newArgument;
+            static SyntaxTrivia GenerateMultiLineComment(SyntaxTrivia singleLineComment)
+            {
+                var commentText = singleLineComment.GetCommentText();
+                return SyntaxFactory.Comment("/* " + commentText + " */");
```

Comment:
comment text may contain `*/`.  you'll need to split that to ensure the comment doesn't terminate too early.  test that.